### ISSUE TITLE
Display in a GUI, in realtime?

### ISSUE THREAD

**Issue by MikeTheWatchGuy** [AgentElement/mandelbrot-set#1]
I haven't downloaded and tried the software, so maybe I'm not correct in how I interpreted the code, poking around.  It looks like you're generating a bunch of individual PNG files and then creating a movie from those.  Is that correct?

If so, then it should be possible to easily display those in a PySimpleGUI based GUI.  There are already a lot of Demo Programs that show in detail how to integrate with a number of packages.

You'll find a YOLO machine learning project for example that displays the images in a GUI.

I'm suggesting the same kind of thing.  Instead of saving the image, you update the GUI window with the new image.

Does this sound like something that would fit into the architecture of your code?

**Comment by AgentElement** [comment]
Yes. That's actually exactly what I'm working on. I'm writing a GUI with PyQt5; this will take a bit more time but hopefully might end up a bit more polished. I'll give PySimpleGUI a try, if it's something I can slap together in a few hours, I'll update the repo with a rudimentary GUI.

**Comment by AgentElement** [comment]
Huh, PySimpleGUI might be exactly what I'm looking for. I'll probably add a GUI in a couple of days, the API is impressively simple.
